src/open_inwoner/scss/components/Header/AnchorMenu.scss

```diff
@@ -82,6 +82,30 @@
     }
   }
 
+  // nested list
+  &__sublist {
```

Comment:
You can use this Sublist class also by setting it as having a default styling of "font-weight" normal when the parent is active/open - and _**only**_ set it to "font-weight: bold" if the heading is actually scrolled to (we do not want everythign to be bold when the user hits only the top-heading).